tests/osquery/test_api_views.py

```diff
@@ -1091,3 +1102,242 @@ def test_export_distributed_query_results_ok(self):
         response = self.post(reverse("osquery_api:export_distributed_query_results", args=(dq.pk,)),
                              include_token=True)
         self.assertEqual(response.status_code, 201)
+
+    # list queries
+
+    def test_get_queries(self):
+        query = self.force_query()
+        self.set_permissions("osquery.view_query")
+        response = self.get(reverse("osquery_api:queries"))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json(),
+                         [{"id": query.pk,
+                           "name": query.name,
+                           "version": 1,
+                           "compliance_check_enabled": False,
+                           "sql": query.sql,
+                           "minimum_osquery_version": None,
+                           "description": query.description,
+                           "value": '',
+                           "platforms": [],
+                           "created_at": query.created_at.isoformat(),
+                           "updated_at": query.updated_at.isoformat()
+                           }])
+
+    def test_get_queries_unauthorized(self):
+        self.set_permissions("osquery.view_query")
+        response = self.get(reverse("osquery_api:queries"), include_token=False)
```

Comment:
The token is not included, no need to add the permissions. This is true for all the unauthorized tests.